utils.js

```diff
@@ -398,8 +398,8 @@ function isDomElement(thing) {
  * execute any of its inline scripts.
  */
 function staticDom(html) {
-    const {jsdom} = require('jsdom');
-    return jsdom(html, {features: {ProcessExternalResources: false}});
+    var parser = new DOMParser();
```

Comment:
The spec (https://w3c.github.io/DOM-Parsing/) says that the default for the text/html type is for script elements to be marked unexecutable. Does this mean the resource is not fetched at all, or just that the code does not get executed?